### PR TITLE
HZ-0201 Create Sign in page layout

### PR DESCRIPTION
**Pull request by Timguau**

### CODE REVIEW

**Review by Kelozheng — APPROVED** [review]

File: src/components/SignIn/SignInInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  width: "100%",
+  height: "50px",
+  marginTop: "10px",
+  backgroundColor: main,
+  "&:hover": {
+    backgroundColor: dark
+  }
+});
+
+const validationSchema = yup.object({
```
Comment (yuanningd) [inline src/components/SignIn/SignInInfo.tsx]: Are you sure this is needed for sign in?

Comment (yuanningd) [reply src/components/SignIn/SignInInfo.tsx]: You can keep the email checked

**Review by jinzhih — APPROVED** [review]

File: src/components/SignIn/SignInInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          onChange={formik.handleChange}
+          onBlurCapture={formik.handleBlur}
+          error={formik.touched.email && Boolean(formik.errors.email)}
+          helperText={formik.touched.email && formik.errors.email}
+        />
+        <SignInInfoTextField
+          required
+          fullWidth
+          id="password"
+          label="Password"
```
Comment (yuanningd) [inline src/components/SignIn/SignInInfo.tsx]: Use Placeholder instead

File: src/components/SignIn/SignInInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  return (
+    <Box width="100%">
+      <form onSubmit={formik.handleSubmit} noValidate>
+        <SignInInfoTextField
+          required
+          fullWidth
+          id="email"
+          type="email"
+          label="Email address"
```
Comment (yuanningd) [inline src/components/SignIn/SignInInfo.tsx]: Same

File: src/components/SignIn/SignInInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+          value={formik.values.email}
+          onChange={formik.handleChange}
+          onBlurCapture={formik.handleBlur}
```
Comment (yuanningd) [inline src/components/SignIn/SignInInfo.tsx]: not needed

File: src/components/SignIn/SignInInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          error={formik.touched.email && Boolean(formik.errors.email)}
+          helperText={formik.touched.email && formik.errors.email}
```
Comment (yuanningd) [inline src/components/SignIn/SignInInfo.tsx]: `formik.touched.email` is not needed

**Review by yuanningd — APPROVED** [review]

**Pull request merged**